docroot/modules/custom/mass_translations/mass_translations.deploy.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/**
+ * @file
+ * Implementations of hook_deploy_NAME() for Mass Translations.
+ */
+
+use Drupal\taxonomy\TermInterface;
+
+/**
+ * Migrate document language fields.
+ */
+function mass_translations_deploy_language_terms(&$sandbox) {
+  $_ENV['MASS_FLAGGING_BYPASS'] = TRUE;
+
+  $english_target_id = 8876;
+
+  $query = \Drupal::entityQuery('media')
+    ->condition('status', 1)
```

Comment:
Unpublished media does not need migration?

Reply:
It does. I added this as a migration. I should note too that I ended up adding a bypass environment variable for a presave function in mass_media (it skips moving files, which was failing the migration when files didn't exist...likely helped a little with performance as well)

Reply:
> Looks nice and clean. IMO, this merits a test or 2 for the key use cases. Ideally we would use the forms to create content and translate it and assert that all looks good along the way.

I also added tests for checking hreflang values on nodes (though I did this for only one content type), checking the Language value on Media pages, and checking that the tabs load on both.